sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -11,6 +11,7 @@ namespace Azure.Containers.ContainerRegistry.Tests
     public class ContainerRepositoryClientLiveTests : RecordedTestBase<ContainerRegistryTestEnvironment>
     {
         private readonly string _repositoryName = "library/hello-world";
+        private readonly string _tagName = "latest";
         private ContainerRepositoryClient _client;
```

Comment:
I haven't seen this pattern in many libraries where a client is initialized as part of setup and then the same instance is used in multiple tests. I suppose it is kind of nice because it tests the thread-safety of individual clients, but in general it is better if each test is as self-contained as possible.

Reply:
Ah, @christothes had suggested this, but perhaps I had misunderstood his suggestion?

Reply:
I've tidied this up in my next PR.  Watch this space ...